content/blog/2022-02-28-using-iterative-terraform-provider-with-docker.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Using Iterative Terraform Provider with Docker
+date: 2022-02-28
+description: >
+  In this tutorial you'll learn how you can move use an existing Docker image with 
+  Iterative Terraform Provider, as well as build your own Docker image.
+descriptionLong: >
+  In this tutorial you'll learn how you can move use an existing Docker image with 
+  Iterative Terraform Provider, as well as build your own Docker image.
+picture: 2022-02-28/unsplash-containers.jpg
+author: maria_khalusova
+#  todo: commentsUrl:
+tags:
+  - MLOps
+  - Iterative Terraform Provider
+  - AWS
+  - Docker
+---
+
+Recently we have [published a tutorial](2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md) 
```

Comment:
Looks like a bad link probably?

Reply:
p.s. another good reason to have a deployed review app, so the automated link checks can run

Reply:
This is a link to the first tutorial that hasn't been published yet.